ui/src/app/metadata/IMetadata.ts

```diff
@@ -0,0 +1,25 @@
+import {ApiJobCode} from "./job-codes/Jobcode"
+import {ApiNamedReference} from "./named-references/NamedReference"
```

Comment:
According to code style in other files, seems we need white space in imports.

Reply:
Correcting..